docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -242,6 +242,100 @@ To keep the same behavior, you can replace it by `hasLeadingZerosInFormat`
  return <DateField unstableFieldRef={fieldRef} />;
 ```
 
+### Headless fields
+
+:::success
+The following breaking changes only impact you if you are using hooks like `useDateField` to build a custom UI.
+
+If you are just using the regular field components, then you can safely skip this section.
```

Comment:
Just an idea when reading it: differences betweeen sections/subsection is not always super clear. This could be a helper. Maybe better to add that kind of stuff after the first stable, when the migration guide structure will stop moving.

```suggestion
If you are just using the regular field components, then you can safely [skip this section](#date-management).
```

Reply:
That's a nice idea
Maybe we can do it on the whole migration guide at the end of the alpha to make sure we don't have outdated things